hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/TestDFSPermission.java

```diff
@@ -32,6 +32,7 @@
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.apache.hadoop.test.LambdaTestUtils;
```

Comment:
the import statements should be sorted.
This should be just after
import org.apache.hadoop.security.UserGroupInformation;